### PR TITLE
[WIP] Test VyOS via Zuul

### PR DESCRIPTION
**Pull request by gundalow**
##### SUMMARY

Add Zuul configuration and roles so that integration tests run for VyOS

* Test via Python 2 and Python 3
* Define aliases `zuul/vyos/group1` and `zuul/network/smoketest`
* Disable Shippable's Network testing as that only covered VyOS
* Only run tests when PRs change the defined `files:` (VyOS code or Zuul
* config)

**Requires:**

- [ ]  https://github.com/ansible/zuul-config/blob/master/resources/ansible.yaml
- [ ]  Software Factory Bot adding to ansible/ansible
##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

**Review by mattclay — APPROVED** [review]

**Review by mattclay — CHANGES_REQUESTED** [review]
Add handling for the `.zuul.d` directory like we have for `.github`:

```
test/runner/lib/classification.py:        if path.startswith('.github/'):
test/runner/lib/target.py:        if root.startswith('.') and root != '.github':
```

That way the yaml files can be sanity tested.

File: shippable.yml [hunk 1]
```diff
@@ -42,8 +42,6 @@ matrix:
     - env: T=windows/2012-R2/4
     - env: T=windows/2016/4
 
-    - env: T=network
-
```
Comment (mattclay) [inline shippable.yml:46]: Lets keep the VyOS testing on Shippable for a while so we have some overlap until we're confident in the results we're getting by running things under Zuul.

**Comment by abenokraitis** [comment]
cc @pabelanger

**Comment by pabelanger** [comment]
I actually think we can close this PR, since it has been created we have evolved to using zuul.ansible.com.  At the moment, we are storing the job configuration in github.com/ansible/ansible-zuul-jobs, but once we enable github app integration for ansible/ansible, we _could_ consider moving that logic in tree. However, we don't really need to do that.

Talking with @mattclay in the testing meeting, we do want to see about doing some integration with current ansiblebot. For that, I still need to look at code and setup check api for network specific testing.

**Pull request closed without merging**